src/Symfony/Component/Intl/Tests/LanguagesTest.php

```diff
@@ -839,6 +1625,12 @@ public function testGetNames($displayLocale)
 
         $this->assertNotEmpty($languages);
         $this->assertEmpty(array_diff($languages, self::$languages));
+
+        foreach ($languages as $alpha2Code) {
+            if (isset(self::$alpha2ToAlpha3[$alpha2Code])) {
+                $this->assertSame(Languages::getAlpha3Name(self::$alpha2ToAlpha3[$alpha2Code], $displayLocale), Languages::getName($alpha2Code, $displayLocale));
```

Comment:
This is not testing the result from `Languages::getNames`.
This is testing `Languages::getAlpha3Name` or `Languages::getName`.

I think each test should be focused on the one method it is declared to test.

Reply:
right now both `getNames` and `getAlpha3Names` are test against each other - see 3ad5d3e1d8554d62e5f9292f290b0d839a4c9212